src/_pytest/_code/code.py

```diff
@@ -188,11 +189,11 @@ def path(self):
         """ path to the source code """
         return self.frame.code.path
 
-    def getlocals(self):
+    @property
```

Comment:
that one is a breaking change of the api

Reply:
I don't think `Code` objects are exposed -- or at least I can't find any cases that they are (?)